### DISCUSSION TITLE
Dynamic Recipients

### DISCUSSION THREAD

**Discussion by AbbeyDesign** [solspace/craft-freeform#382] [Questions & Answers]
How does one add dynamic recipients to the cc setting in a notification template? Every approach I have tried results in the following error:

Address in mailbox given [theEmails] does not comply with RFC 2822, 3.6.2.

**Comment by kjmartens** [comment]
Hi @AbbeyDesign,

There isn't a way to do this currently. It's a neat idea though, so I'll make a note about this for the feature requests list for future consideration. 🙂

**Reply by AbbeyDesign** [reply]
So apparently this turned out to be just an email field so I am trying a workaround. the {# cc #} doesn't seem to take anything set earlier.

If I do this:
{% set theEmails = ANY COMBINATIONS OF TWIG OR HARDCODED EMAILS %}
and then do 
{# cc theEmails #}
I get:
Address in mailbox given [thex] does not comply with RFC 2822, 3.6.2.
